Resolved Xbox bug 37316753

v-dekennedy opened the pull request:
Updated two screenshots to resolve bug 37316753.

v-dekennedy committed:
Updated TitleID screenshot

v-dekennedy committed:
Updated My Studios and Titles screenshot

ghost commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/signed)](https://cla.opensource.microsoft.com/MicrosoftDocs/playfab-docs?pullRequest=228) <br/>All CLA requirements met.

v-dekennedy committed:
Update quickstart.md

Updated description of screenshot so that the title and TitleId properly match.

v-dekennedy committed:
Update quickstart.md

Found & corrected typos, changing "Game Manger" to "Game Manager".
